candi/openapi/doc-ru-node_group.yaml

```diff
@@ -99,6 +99,8 @@ spec:
                       description: "Status of the condition summary."
             spec:
               properties:
+                fastShutdown: &fastShutdown
```

Comment:
Nitpicking: it's rather 'quick' than 'fast'

Reply:
Indeed, it is.